### PR TITLE
Inconsistency between website documentation and tutorials

### LINKED ISSUE #88: inconsistent wording between website docs and tutorials
See http://stackoverflow.com/questions/32906938/rabbitmq-acks-republishing

### PR DESCRIPTION
**Pull request by asbjornu**
Made the wording between the documentation (in semantics.xml) and tutorials regarding redelivering of messages consistent. Fixes #88.

### CODE REVIEW

File: site/tutorials/tutorial-two-dotnet.md [hunk 1]
```diff
@@ -213,9 +213,9 @@ message wasn't processed fully and will redeliver it to another
 consumer. That way you can be sure that no message is lost, even if
 the workers occasionally die.
 
-There aren't any message timeouts; RabbitMQ will redeliver the message
-only when the worker connection dies. It's fine even if processing a
-message takes a very, very long time.
+There aren't any message timeouts; RabbitMQ will redeliver the message when
+the consumer dies. It's fine even if processing a message takes a very, very
```
Comment (michaelklishin) [inline site/tutorials/tutorial-two-dotnet.md]: This is not specific enough. Channel closure (or connection loss) would result in messages re-queueing. Which is what the previous version tries to say.

Comment (asbjornu) [inline site/tutorials/tutorial-two-dotnet.md]: Ok, I'm not familiar with the technical details, I just saw #88 and the [accepted answer on Stack Overflow](http://stackoverflow.com/a/32909897/61818) which has the wording I've pull requested. Should #88 just be closed as `wontfix`, then?

**Comment by michaelklishin** [comment]
@asbjornu no but we need to try to explain what "consumer dies" means: channel or connection closure, including TCP connection. This is a matter of wording, I'm not questioning the premise of #88.

**Comment by michaelklishin** [comment]
@asbjornu if you don't feel confident about working on the wording, I can do that.

**Comment by michaelklishin** [comment]
@asbjornu ok, re-worded and deployed.

You've been contributing increasingly useful changes. Can you please file a signed [contributor agreement](https://github.com/rabbitmq/ca), as described in [the README](https://github.com/rabbitmq/ca)? It would greatly help us in accepting more contributions from you in the future :)

Thank you!

**Comment by asbjornu** [comment]
@michaelklishin Thanks. The new wording is indeed better. I'll sign the agreement as soon as I can get to it. :)

**Pull request merged**